examples/shortest_path_follower_example.py

```diff
@@ -35,7 +35,9 @@ def get_info(self, observations):
 
 
 def draw_top_down_map(info, heading, output_size):
-    top_down_map = maps.colorize_topdown_map(info["top_down_map"]["map"])
+    top_down_map = maps.colorize_topdown_map(
+        info["top_down_map"]["map"], info["top_down_map"]["fog_of_war_mask"]
```

Comment:
As I understand functionality be tested as fog became enabled by default. Curious if it doesn't fail when `TASK.TOP_DOWN_MAP.FOG_OF_WAR.DRAW = FALSE`.

Reply:
`info["top_down_map"]["fog_of_war_mask"]` is `None` when `TASK.TOP_DOWN_MAP.FOG_OF_WAR.DRAW = FALSE` and the logic in `maps.colorize_topdown_map` defaults to the old behavior when the fog-of-war mask is None, so it works :) (I have also tested this).